### ISSUE TITLE
xoai documentation for 4.1.x

### ISSUE THREAD

**Issue by dloy** [lyncode/xoai#35]
I would like to implement a standalone oai-pmh version 2 data provider server using xoai 4.1.x without DSpace.

Can anyone direct me to any documentation for doing this or to a reference implementation? 
Thanks David

**Comment by helix84** [comment]
I don't think there's any documentation yet, but it might help to take a look at another project that's based on xoai:

https://github.com/lyncode/OAIProxySource
https://github.com/lyncode/OAIProxy

**Comment by Chort409** [comment]
Yeah, it would be nice to have some documentation... At least some details in https://github.com/DSpace/xoai/wiki/How-to-Implement would be nice to have

**Comment by tdonohue** [comment]
Documentation contributions are welcome.  Recently, this "xoai" project was transitioned from https://github.com/lyncode/ to https://github.com/DSpace/.   It came with the little provided documentation, and no one has been able to enhance the docs (we're still learning the ins and outs of the codebase ourselves).

So, contributions of documentation are more than welcome.

**Comment by Chort409** [comment]
Good to know, if I would be able to use xoai code in my project I will try to come up with something...
It looks like somebody started to write [How-to-Implement](https://github.com/DSpace/xoai/wiki/How-to-Implement) and then was hit by a bus :)

**Comment by Chort409** [comment]
It would be very handy if you could add a dummy implementation of AOI data provider with just a few hardcoded items inside... With several comments like "YOUR CODE GOES HERE". Currently there is almost 0 comments in the code and DSpace itself is using 3.x version, so it is difficult to use it as the reference.

**Comment by kprasha** [comment]
are there any documentations now? asking as the above discussion is quite old and there is a possibility of new documentation.

**Comment by tdonohue** [comment]
Unfortunately, this module has not been a high priority for the DSpace project in recent years.  The module is still used by DSpace, but has been stable for our purposes. So, we still have not found anyone interested in contributing to the documentation.  Again though, contributions to documentation are welcome.
